README.md

```diff
@@ -161,6 +161,7 @@ See also [Internet of Things (IoT)](#internet-of-things-iot)
 - [Alltube](http://www.alltubedownload.net) - Web interface for [youtube-dl](https://github.com/rg3/youtube-dl), a program to download videos and audio from [more than 100 websites](https://rg3.github.io/youtube-dl/supportedsites.html). ([Source Code](https://github.com/Rudloff/alltube)) `GPL-3.0` `PHP`
 - [AmIUnique](https://amiunique.org/) - Learn how identifiable you are on the Internet (browser fingerprinting tool). ([Source Code](https://github.com/DIVERSIFY-project/amiunique)) `MIT` `Java`
 - [Beehive](https://github.com/muesli/beehive) - Flexible event and agent system, which allows you to create your own agents that perform automated tasks triggered by events and filters. `AGPL-3.0` `Go`
+- [betanin](https://github.com/sentriz/betanin/) - Music organization man-in-the-middle of your torrent client and music player. Based on [beets.io](https://beets.io), similar to [Sonarr](https://sonarr.tv/) and [Radarr](https://radarr.video/). `GPL-3.0` `Python`
```

Comment:
Please remove inline links to beets/sonarr/radarr (you can keep the text, just remove the markdown links)